src/providers/twitch/eventsub/MessageBuilder.cpp

```diff
@@ -166,4 +198,84 @@ void makeModerateMessage(
     builder->timeoutUser = action.userLogin.qt();
 }
 
+void makeModerateMessage(
+    EventSubMessageBuilder &builder,
+    const lib::payload::channel_moderate::v2::Event &event,
+    const lib::payload::channel_moderate::v2::Followers &action)
+{
+    QString duration;
+    if (action.followDurationMinutes > 0)
+    {
+        duration = u"(%1 minutes)"_s.arg(action.followDurationMinutes);
+    }
+    makeModeMessage(builder, event, u"followers-only"_s, true, duration);
+}
+void makeModerateMessage(
+    EventSubMessageBuilder &builder,
+    const lib::payload::channel_moderate::v2::Event &event,
+    const lib::payload::channel_moderate::v2::FollowersOff & /*action*/)
+{
+    makeModeMessage(builder, event, u"followers-only"_s, false);
+}
+
+void makeModerateMessage(
+    EventSubMessageBuilder &builder,
+    const lib::payload::channel_moderate::v2::Event &event,
+    const lib::payload::channel_moderate::v2::EmoteOnly & /*action*/)
+{
+    makeModeMessage(builder, event, u"emote-only"_s, true);
+}
+void makeModerateMessage(
+    EventSubMessageBuilder &builder,
+    const lib::payload::channel_moderate::v2::Event &event,
+    const lib::payload::channel_moderate::v2::EmoteOnlyOff & /*action*/)
+{
+    makeModeMessage(builder, event, u"emote-only"_s, false);
+}
+
+void makeModerateMessage(EventSubMessageBuilder &builder,
+                         const lib::payload::channel_moderate::v2::Event &event,
+                         const lib::payload::channel_moderate::v2::Slow &action)
+{
+    makeModeMessage(builder, event, u"slow"_s, true,
+                    u"(%1 seconds)"_s.arg(action.waitTimeSeconds));
+}
+void makeModerateMessage(
+    EventSubMessageBuilder &builder,
+    const lib::payload::channel_moderate::v2::Event &event,
+    const lib::payload::channel_moderate::v2::SlowOff & /*action*/)
+{
+    makeModeMessage(builder, event, u"slow"_s, false);
+}
+
+void makeModerateMessage(
+    EventSubMessageBuilder &builder,
+    const lib::payload::channel_moderate::v2::Event &event,
+    const lib::payload::channel_moderate::v2::Subscribers & /*action*/)
+{
+    makeModeMessage(builder, event, u"subscribers-only"_s, true);
+}
+void makeModerateMessage(
+    EventSubMessageBuilder &builder,
+    const lib::payload::channel_moderate::v2::Event &event,
+    const lib::payload::channel_moderate::v2::SubscribersOff & /*action*/)
+{
+    makeModeMessage(builder, event, u"subscribers-only"_s, false);
+}
+
+void makeModerateMessage(
+    EventSubMessageBuilder &builder,
+    const lib::payload::channel_moderate::v2::Event &event,
+    const lib::payload::channel_moderate::v2::Uniquechat & /*action*/)
+{
+    makeModeMessage(builder, event, u"r9k"_s, true);
+}
+void makeModerateMessage(
+    EventSubMessageBuilder &builder,
+    const lib::payload::channel_moderate::v2::Event &event,
+    const lib::payload::channel_moderate::v2::UniquechatOff & /*action*/)
+{
+    makeModeMessage(builder, event, u"r9k"_s, false);
+}
```

Comment:
Let's call it unique-chat - it's what the feature's new name is
```suggestion
void makeModerateMessage(
    EventSubMessageBuilder &builder,
    const lib::payload::channel_moderate::v2::Event &event,
    const lib::payload::channel_moderate::v2::Uniquechat & /*action*/)
{
    makeModeMessage(builder, event, u"unique-chat"_s, true);
}
void makeModerateMessage(
    EventSubMessageBuilder &builder,
    const lib::payload::channel_moderate::v2::Event &event,
    const lib::payload::channel_moderate::v2::UniquechatOff & /*action*/)
{
    makeModeMessage(builder, event, u"unique-chat"_s, false);
}
```